docs/grid_model/network_subnetwork.md

```diff
@@ -807,6 +807,7 @@ A busbar section is a non impedant element used in a node/breaker substation top
 ## Breaker/switch
 
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Switch.html)<br>
+A switch to connect a piece of equipment in a substation.
```

Comment:
A switch is a device that interrupts currents in a circuit. It can either allow current to flow when closed or stop it when open. There are several types of switches: breakers, that are capable of opening fault currents, and disconnectors or load break switches that are not capable of.

Reply:
@jeandemanged do you agree?